README.md

```diff
@@ -2,3 +2,68 @@
 
 A DAOstack product
 
+## Getting Started
+For Getting Started with React Native there is a very comprehensive guide on their website
+
+https://reactnative.dev/docs/getting-started
+
+There are two documentation options: Expo CLI Quickstart & React Native CLI Quickstart.
+
+Choose the React Native CLI Quickstart and then choose whether you use a Mac, Linux or Windows
+
+### Files containing keys
+You will need the `env.json` file from one of the members of the developer team to get started.
+This file is to be placed into the root folder of the project.
+
+### Getting Started from the Common Github Repo
+
+1. Clone the repo `git clone https://github.com/daostack/common.git`
+2. `cd common` and run `yarn` or `npm install`
+3. `cd ios` and run `pod install`
+
+### For iOS
+
+4. To run the project from the command line run `react-native run-ios`
+5. To run from Xcode, open the `common.xcworkspace` file and then enter `command + r`
+
+### For Android
+
+6. Run an Android simulator device
+7. Enter `react-native run-android` in the command line
```

Comment:
Hey @markspereira  does not work on Ubuntu linux (at least not on my machine)

What I did:

 `npm run start`  to start the metro bundle 
`npm run android` (in another shell) to start the emulator

which works